### PR TITLE
Add testing tip to calendar-walkthrough.md

### PR DESCRIPTION
**Pull request by josueetcom**
The testing tip provides instructions for how to run a specific test method, which would is helpful in ensuring a particular test case passes and removes the need to sift through test output for other tests.

### CODE REVIEW

File: walkthroughs/week-4-tdd/project/calendar-walkthrough.md [hunk 1]
```diff
@@ -73,6 +73,11 @@ mvn test
 
 When all the tests pass, you can be confident that your code works!
 
+**Tip**: You execute a single test method by passing the flag`-Dtest=FindMeetingQueryTest#testMethodName`. E.g. To run just the test method named `optionsForNoAttendees()` you would execute:
```
Comment (KevinWorkman) [inline walkthroughs/week-4-tdd/project/calendar-walkthrough.md]: Can you add a space here, between the g and the backtick?

Comment (KevinWorkman) [inline walkthroughs/week-4-tdd/project/calendar-walkthrough.md]: Can you add a newline to fix the line wrapping?

Comment (josueetcom) [reply walkthroughs/week-4-tdd/project/calendar-walkthrough.md]: Done

Comment (josueetcom) [reply walkthroughs/week-4-tdd/project/calendar-walkthrough.md]: Done

**Pull request merged**